### ISSUE TITLE
Do not see Aria in the Setting

### ISSUE THREAD

**Issue by Linhtinh123** [lifan0127/ai-research-assistant#130]
Hi Mr Fan Li,

I am using Zotero 7 and installed the updated Aria. 
But I cannot see it the the setting to add API.

![image](https://github.com/user-attachments/assets/8df00df8-87ad-4b65-837f-720b675a0551)

**Comment by lifan0127** [comment]
Hi @Linhtinh123 Sorry for the delay. It seems the plugin was not loaded successfully. Are you using the latest Zotero 7 beta version? Could you please try temporarily disable other plugins and see if it works? Thanks

**Comment by Linhtinh123** [comment]
yes, I am using Zotero 7 and Aria works when I disable other plugins. 
Does it mean I always have to disable other plugins to use Aria?

**Comment by lifan0127** [comment]
Thanks! Can I trouble you to enable other plugins one by one so we can find out which one causes the conflict? Then I may be able to help you troubleshoot further.

**Comment by lifan0127** [comment]
Hi @Linhtinh123 Could you please try the following release. I believe the compatibility issue has been fixed. Thanks!

https://github.com/lifan0127/ai-research-assistant/releases/tag/v0.7.4
